PokeAlarm/Manager.py

```diff
@@ -981,8 +981,11 @@ def process_egg(self, egg):
 
         # Check if egg gym filter has a contains field and if so check it
         if len(self.__egg_settings['contains']) > 0:
-            if not any(x in gym_info['name'].lower() for x in self.__egg_settings['contains']):
-                log.info("Egg {} ignored: gym name did not match contains filter.".format(gym_info['name'].lower()))
+            if not any(x in gym_info['name'].lower()
+                       for x in self.__egg_settings['contains']):
```

Comment:
Have you tested this? It looks like your indentation might be off, and only the first gym_name contains will be checked